deploy/files/scripts/setup_db_replication.sh

```diff
@@ -24,7 +24,7 @@ psql "$DB" -U "$DB_USER" -c 'create extension postgis;'
 source /var/www/polling_stations/code/venv/bin/activate
 
 # Migrate db - this builds the schema before syncing
-IGNORE_ROUTERS=True /var/www/polling_stations/code/manage.py migrate --database=local
+/var/www/polling_stations/code/manage.py migrate --database=local
```

Comment:
I have also removed this here because it doesn't do anything.

`IGNORE_ROUTERS` was removed in https://github.com/DemocracyClub/UK-Polling-Stations/commit/f163bba0be12a1cfd155d1bdeb3c7bd61b19f63d so this is just a bit of cleanup